packages/@aws-cdk/aws-ecs/lib/log-drivers/generic-log-driver.ts

```diff
@@ -0,0 +1,62 @@
+import { Construct } from '@aws-cdk/core';
+import { ContainerDefinition } from '../container-definition';
+import { LogDriver, LogDriverConfig } from "./log-driver";
+
+/**
+ * A log driver that sends log information to CloudWatch Logs.
```

Comment:
This description doesn't sound like it's true if you *must* use AwsLogDriver to use `awslogs`.  Or is it?